piece.py

```diff
@@ -17,7 +15,8 @@ def get_current_pos(self):
         return self.his_pos[-1]
 
     def __str__(self):
-        return self.nick_name
-
+        x, y = self.get_current_pos()
+        #return self.name + ': (' + str(x) + ', ' + str(y) + ')'
+        return self.type[0:3]
```

Comment:
def __str__(self):
        return self.nick_name
would be better in my opinion